File: mario.c
#include <stdio.h>
#include <cs50.h>

int get_size(void);
void print_grid(int size);

int main(void)
{
    // get user input
    int n = get_size();

    // print grid of bricks
    print_grid(n);
}

int get_size(void)
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 0);

    return n;
}

void print_grid(int size)
{
    for (int i = 0; i < size; i++)
    {
        for k 
        for (int j = 0; j < size; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: world-cup/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

n = 0

while n < 1 or n > 8:
    n = get_int("Height: ")

for i in range(n):
    for k in range(n - (i + 1)):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print(end="  ")
    for m in range(i + 1):
        print("#", end="")
    print()